src/main/java/seedu/address/logic/parser/ArgumentTokenizer.java

```
@@ -119,9 +134,46 @@ private static String extractArgumentValue(String argsString,
         Prefix prefix = currentPrefixPosition.getPrefix();
 
         int valueStartPos = currentPrefixPosition.getStartPosition() + prefix.getPrefix().length();
-        String value = argsString.substring(valueStartPos, nextPrefixPosition.getStartPosition());
+        String value = argsString.substring(valueStartPos, nextPrefixPosition.getStartPosition())
+            .trim()
+            .replace("\\\"", "\""); // remove backslash from escaped double-quote
+
+        value = value.trim();
+        if (value.isEmpty()) {
+            return value;
+        }
+
+        // Remove double-quote escape from the actual value
+        if (value.charAt(0) == '"' && value.charAt(value.length() - 1) == '"') {
+            value = value.substring(1, value.length() - 1);
+        }
 
-        return value.trim();
+        return value;
```

Does this correctly handle the case where a user enters a value like n/"Name1""Name2"?
 I hypothesis that it will make the name Name1""Name2. Do we accept this or should we throw an error instead?